modules/spacecraftinstruments/rendering/renderablemodelprojection.cpp

```diff
@@ -184,6 +241,14 @@ ghoul::opengl::Texture& RenderableModelProjection::baseTexture() const {
 }
 
 void RenderableModelProjection::render(const RenderData& data, RendererTasks&) {
+    // Update boundingsphere
+    setBoundingSphere(_geometry->boundingRadius() * _modelScale *
+        glm::compMax(data.modelTransform.scale)
+    );
+
+    // Set Interaction sphere size to be 10% of the bounding sphere
+    setInteractionSphere(_boundingSphere * 0.1);
```

Comment:
Do we really need to do this every frame? Same for bounding sphere

Reply:
This is being done every frame now to account for the scale we can change in the GUI. But maybe we don't need to account for that every frame?